model/task/task.go

```diff
@@ -1053,7 +1053,16 @@ func UnscheduleStaleUnderwaterTasks(distroID string) (int, error) {
 		},
 	}
 
-	info, err := UpdateAll(query, update)
+	// Force the query to use 'distro_1_status_1_activated_1_priority_1'
+	// instead of defaulting to 'status_1_depends_on.status_1_depends_on.unattainable_1'.
+	hint := bson.D{
```

Comment:
Instead of putting this within the function, can we define this as a constant? (Maybe in db.go) Then we can also use it in the test

Reply:
done